src/content/pages/thank-you.mdx

```diff
@@ -0,0 +1,457 @@
+---
+title: Thank You
+subtitle: For the EuroPython 2024 Team
+---
+
+# Thank You
+
+As we reflect on EuroPython 2024 in Prague, we are filled with immense gratitude for the year-long journey marked by innovation, teamwork, and dedication. Our incredible team infused the event with their unique blend of enthusiasm, kindness, and creativity! We extend our heartfelt thanks to everyone who contributed to making EuroPython 2024 a memorable experience both in Prague and online.
+
+Since 2002, EuroPython has thrived on the passion and hard work of volunteers. This year was no exception, with our devoted team investing countless hours and boundless energy into bringing EuroPython 2024 to life. Let us now celebrate the amazing individuals whose efforts made this event possible!
+
+
+# EuroPython Society Board
+
+The annually elected EuroPython Society (EPS) Board manages the day-to-day operations of the EPS, the legal entity behind the EuroPython conference series. The EPS 2024 Board were responsible for most of the fiscal, legal and venue related work for the EuroPython 2024 conference, including venue & supplier selections and contract negotiations.
+
+- Alexander Hendorf
+- Anders Hammarquist (Treasurer)
+- Artem Kislovskiy (Vice-Chair)
+- Artur Czepiel (Chair)
+- Laís Carvalho
+- Sangarshanan Veera
+- Sebastiaan Zeeff
+- Theofanis Petkos
+- Vaibhav (VB) Srivastav
+
+
+# Code of Conduct Committee
+
+These humans are responsible for receiving & handling Code of Conduct reports and help the EuroPython community remain a safe and friendly place:
+
+- Cheuk Ting Ho
+- Jakub Vysoky
+- Lumír Balhar
+- Naomi Ceder
+
```

Comment:
```suggestion
Vicky Twomey-Lee

```
Vicky is part of the CoC team even though she could not physically be there. So I think she should be listed.